Textile/Classes/CafesApi.m

```diff
@@ -16,6 +16,9 @@ - (void)register:(NSString *)host token:(NSString *)token error:(NSError * _Null
 
 - (CafeSession *)session:(NSString *)peerId error:(NSError * _Nullable __autoreleasing *)error {
   NSData *data = [self.node cafeSession:peerId error:error];
+  if(!data) {
+    return nil;
+  }
```

Comment:
We have to explicitly check if the data is nil and return nil if so. Otherwise, a default `CafeSession` will be created and returned when there was no session actually found.